packages/dds/tree/src/simple-tree/types.ts

```diff
@@ -38,6 +38,69 @@ import { tryGetSchema } from "./treeNodeApi.js";
  */
 export type Unhydrated<T> = T;
 
+export interface NodeChangedEventArgs {
+	readonly changedProperties: ReadonlySet<string>;
+}
+
+export interface TreeChangeEventsBase {
```

Comment:
`TreeChangeEventsBase` keeps the common events that look the same regardless of node kind.